engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/migration/MigrationExecutionDto.ftl

```diff
@@ -0,0 +1,49 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/migration/execute-migration/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto>
+    
+    <@lib.property
+        name = "migrationPlan"
+        type = "object"
+        dto = "MigrationPlanDto"
+        additionalProperties = true
+        desc = "The migration plan to execute. A JSON object corresponding to
+                the migration plan interface in the engine as explained below."
+    />
+
+    
+    <@lib.property
+        name = "processInstanceIds"
+        type = "array"
+        desc = "A list of process instance ids to migrate."
+    />
+
+    
+    <@lib.property
+        name = "processInstanceQuery"
+        type = "object"
+        dto = "ProcessInstanceQueryDto"
```

Comment:
Same as the `MigrationPlanDto`. There is only a single instance of `ProcessInstanceQueryDto`. A property of type `object` will be converted to a Java Map.

I think it should be the following:
```suggestion
        type = "ref"
        dto = "ProcessInstanceQueryDto"
```

You can check out the `ProcessInstanceSuspensionStateAsyncDto.ftl` file, I think we have the same case there.